lrauv_ignition_plugins/worlds/portuguese_ledge.py

```diff
@@ -0,0 +1,17 @@
+# Takes in a template and generates an SDF file.
+
+import em
+import os
+import sys
+
+def main():
+    in_file = 'portuguese_ledge.sdf.em'
+    out_file = 'portuguese_ledge.sdf'
```

Comment:
@chapulina consider resolving these paths w.r.t. the directory containing this script. Better for scripts to be workspace agnostic IMO.

Reply:
I did this initially, but when I changed it to autogenerate during compilation, I reverted to using absolute paths.